README.md

```diff
@@ -68,29 +68,34 @@ brew install k6
 
 ### Windows
 
-You can manually download and install the [official `.msi` installation package](https://dl.bintray.com/loadimpact/windows/k6-v0.31.1-amd64.msi) or, if you use the [chocolatey package manager](https://chocolatey.org/), follow [these instructions](https://bintray.com/repo/buildSettings?repoPath=%2Floadimpact%2Fchoco) to set up the k6 repository.
+If you use the [Chocolatey package manager](https://chocolatey.org/):
 
-### Linux
+```
+choco install k6
+```
 
-**Notice: Because [Bintray is being shutdown](https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/) we are going to start self-hosting our packages soon, before k6 v0.32.0. This means you will have to re-install them, since the old .rpm and .deb repos will stop working.**
+Otherwise you can manually download and install the [latest official `.msi` package](https://dl.k6.io/msi/k6-latest-amd64.msi).
 
-For Debian-based Linux distributions, you can install k6 from the private deb repo like this:
+### Linux
+
+For Debian-based Linux distributions like Ubuntu, you can install k6 from the private deb repo like this:
 
 ```bash
-sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 379CE192D401AB61
-echo "deb https://dl.bintray.com/loadimpact/deb stable main" | sudo tee -a /etc/apt/sources.list
+curl -s https://dl.k6.io/key.gpg | sudo apt-key add -
```

Comment:
I don't think we should bother uploading the key to any keyservers, as they're usually slow and flaky. However, we lose any potential web of trust benefits, and we're not specifying a key ID here, which could lead to users installing the wrong key if `dl.k6.io` is compromised.

Any thoughts on a middle ground solution?

Reply:
Hmm I don't have a problem with uploading the key to a keyserver or two, though I'm not sure if we need to request users to explicitly install it? Doesn't `apt-get install k6` ask users to trust our key the first time they try to install it? If so, we can just add a comment like this:
> apt-get will ask you to trust our key <keyid>, confirm it or download it from https://dl.k6.io/key.gpg and import it

Reply:
> Doesn't `apt-get install k6` ask users to trust our key the first time they try to install it?

No, apt is not that smart :sweat:, and fails at `apt-get update` if the key doesn't exist:

```
Err:1 https://dl.k6.io/deb stable InRelease
  The following signatures couldn't be verified because the public key is not available: NO_PUBKEY C780D0BDB1A69C86
```

But I added the key to `keyserver.ubuntu.com` and `pgp.mit.edu`, so I'm using `--recv-keys` in the latest push, with a fallback to `curl | apt-key add` if that fails.